### ISSUE TITLE
Review label for OBI_0003005 'device for resting of adult arthropods for collection'

### ISSUE THREAD

**Issue by bobular** [VEuPathDB-ontology/VEuPathDB-ontology#180]
Definition is fine: "A device that is designed to provide a solid surface that is attractive to an arthropod as a place to rest during a period of inactivity."

I found the label hard to parse.

label could be "device for collection of resting adult arthropods" ?

http://www.ontobee.org/ontology/OBI?iri=http://purl.obolibrary.org/obo/OBI_0003005

**Comment by cstoeckert** [comment]
Change was made in latest OBI release
